src/index.md

```diff
@@ -1 +1,81 @@
-# What is Time?
+# Introduction
+
+`time` is a date and time library for Rust. It's:
+
+- **Easy and safe**. `time` has a [straightforward API](https://time-rs.github.io/api/time/struct.OffsetDateTime.html) without footguns.
+- **space optimal and efficient**. `time` provides support for dates in the ±9999 year range, with nanosecond precision; ranges up to ±999,999 are supported with the `large-dates` feature.
+- **`serde` ready**. Supports ISO8601, RFC2822 and RFC3339 with the `serde-well-known` feature. Not in the list? [Make your format](./api/format-description.md).
+- **`const` ready**. A majority of the API is `const`, making it ready for resource-constrained applications, with optional [macros](https://time-rs.github.io/api/time/macros/index.html) for easy date creation. 
+- **`no-std` support** with `alloc` and `std` features.
+- **numeric traits**. Use durations easily: `2.seconds()`.
+- Supports Windows, Linux, macOS, [WebAssembly](https://developer.mozilla.org/fr/docs/WebAssembly) targets among others.
+- Six-month [minimum supported Rust version](https://rust-lang.github.io/rfcs/2495-min-rust-version.html) guarantee.
+
+And more...
+
+## Getting started 
+
+This short tutorial describes basic usage of `time`, to get operational quickly.
+
+1. **Install `time`**.  Add it to your `Cargo.toml`. We'll enable `macros`:
+```toml
+[dependencies]
+time = { version = "0.3", features = ["macros"] }
+```
+
+2. **Create dates and times.** We can create dates ([`Date`](https://docs.rs/time/latest/time/struct.Date.html)), dates with times ([`PrimitiveDateTime`](https://docs.rs/time/latest/time/struct.PrimitiveDateTime.html)) and date times with an UTC offset ([`OffsetDateTime`](https://docs.rs/time/latest/time/struct.OffsetDateTime.html)). A simple [`Time`](https://docs.rs/time/latest/time/struct.Time.html) is also available.
```

Comment:
This line (and any others that are sufficiently long) should be wrapped to 100 characters.